help/sources/connectors/marketing-automation/pathfactory.md

```diff
@@ -0,0 +1,68 @@
+---
+title: PathFacory Source Overview
+description: Learn how to connect PathFactory to Adobe Experience Platform using APIs or the user interface.
+last-substantial-update: 2024-04-20
+badge: Beta
+exl-id: 
+---
+# [!DNL PathFactory]
+
+>[!NOTE]
+>
+>The [!DNL PathFactory] source is in beta. Please read the [sources overview](../../home.md#terms-and-conditions) for more information on using beta-labeled sources.
+
+Adobe Experience Platform allows data to be ingested from external sources while providing you with the ability to structure, label, and enhance incoming data using Platform services. You can ingest data from a variety of sources such as Adobe applications, cloud-based storage, databases, and many others.
+
+Experience Platform provides support for ingesting data third-party marketing automation system. Support for marketing automation providers includes [!DNL PathFactory].
+
+[[!DNL PathFactory]](https://www.pathfactory.com/) offers a cloud-based platform that helps businesses manage content journeys and drive engagement through intelligent content insights. This guide details how you can integrate data from PathFactory into your Experience Platform, utilizing PathFactory's connectors for optimal data ingestion.
```

Comment:
Revise to: "...from [!DNL PathFactory] into Experience Platform, while utilizing [!DNL PathFactory]'s connectors for optimal data ingestion."

Reply:
Fixed